benchmarks/user/benchmark.py

```diff
@@ -87,7 +87,8 @@ def from_value(ctx, param, value):
         click.option('--dle', callback=from_value,
                      type=click.Choice(['noop'] + configuration._accepted['dle']),
                      help='Devito loop engine (DLE) mode'),
-        click.option('-a', '--autotune', is_flag=True,
+        click.option('-a', '--autotune', default='aggressive',
+                     type=click.Choice(['basic','aggressive']),
```

Comment:
Note for reviewers: This is a reminder to take care about --autotune in bench and plotting that is probably obsolete.
Also to take care about the default values of autotuning that are attached to an operator.
The problem is that after a 'bench', user cannot do a 'plot' with same Operator arguments.